exec/java-exec/src/main/java/org/apache/drill/exec/server/options/SystemOptionManager.java

```diff
@@ -314,6 +344,19 @@ public void deleteAllOptions(OptionType type) {
     }
   }
 
+  public void populateDefaultValues() {
+
+    // populate the options from the config
+    final Map<String, OptionValidator> tmp = new HashMap<>();
+    for (OptionValidator validator: VALIDATORS.values()) {
+      try {
+         validator.loadDefault(bootConfig);
+      } catch (ConfigException.Missing e) {
+        throw new IllegalStateException("Config Option is missing"+e+ validator.getOptionName());
```

Comment:
`...missing " + validator.getOptionName()`
The exception adds no real value here. Or:
`new IllegalStateException(..., e);`